metals/src/main/scala/scala/meta/internal/metals/MetalsServerConfig.scala

```diff
@@ -133,7 +133,6 @@ object MetalsServerConfig {
       case "coc.nvim" =>
         base.copy(
           statusBar = StatusBarConfig.showMessage,
-          isHttpEnabled = true,
```

Comment:
I've taken this out and there are also some other changes that I'll make to the configuration like `isInputBoxEnabled` being turned on and `executeClientCommand` also being on for `coc-metals`. However, I'm turning those on in the extension itself in order to allow someone to still bootstrap Metals themselves for coc.nvim and not use the extension. If they do that, then client commands and input boxes won't be implemented allowing the default settings in here to still take effect.

Reply:
Should we introduce a `-Dmetals.client=coc-metals` instead? There might still be coc.nvim users that are not using coc-metals.

Reply:
I've thought about that, and I would say that if it moves away any further, that might be a good idea. For now, I think just setting the extra needed values in `coc-metals` should suffice, and then that way we don't have to add another entry for it. Using coc.nvim without coc-metals should still work fine with what is currently in here.

Maybe once I dig into the tree view stuff, then it'll be a good idea to split it out.

Reply:
Thinking about it more, I'll just do it right away. I'll end up doing it anyways, so why not now.

Reply:
See my other comment about not doing this now.